apps/i18n/common/en_us.json

```diff
@@ -573,6 +573,8 @@
   "instructions": "Instructions",
   "joinASection": "Join a section",
   "joinSection": "Join section",
+  "joinSectionExplination": "Ask your students to join your section by going to this link and entering the section code (above): ",
```

Comment:
Not sure what a better way to localize these strings would be - I've just made it so the links are at the end and generated in JS.

Reply:
We've run into similar problems with links in the context of strings, and this is consistent with solutions I've seen in the past.